content/_data/changelogs/lts.yml

```diff
@@ -5298,6 +5298,334 @@
       Upgrade from xstream 1.4.15 to 1.4.16.
 
+- version: "2.289.1"
+  date: 2021-06-02
+  lts_predecessor: "2.277.4"
+  lts_baseline: "2.289"
+  changes: # compared to lts_baseline 2.289 - extracted from the RC commit(s)
+
+  - type: major bug
+    category: regression
+    pull: 5489
+    issue: 65605
+    authors:
+    - jtnord
+    message: |-
+      Fix <code>NoSuchMethodError</code> when using plugins that rely on bridge methods for compatibility (regression in 2.278).
+
+  - type: bug
+    category: regression
+    issue: 65585
+    authors:
+    - res0nance
+    message: |-
+      Fix form submission for some specific form validation cases (regression in 2.289).
+
+  - type: bug
+    category: bug
+    pull: 5448
+    issue: 65190
+    authors:
+    - benebsiny
+    message: |-
+      Wrap the build name in the build results list if it is too long.
+
+  - type: bug
+    category: bug
+    pull: 5455
+    issue: 64294
+    authors:
+    - jglick
+    message: |-
+      Improve performance for standard input of the Jenkins CLI, for example with the `install-plugin` command.
+
+  - type: bug
+    category: bug
+    pull: 5435
+    issue: 52356
+    authors:
+    - basil
+    message: |-
+      Fix an issue archiving files greater than 4 GiB in size when creating ZIP64 archives.
+
+  lts_changes: # compared to lts_predecessor 2.277.4 (selected by personal review)
+
+  - type: major rfe
+    category: major rfe
+    pull: 5049
+    issue: 64107
+    authors:
+    - kuisathaverat
+    - timja
+    references:
+      - issue: 55582
+      - issue: 64107
+      - pull: 5049
+      - pull: 5206
+      - url: https://github.com/jenkinsci/sshd-module
+        title: SSHD module repository
+    message: |-
+      SSHD module is no longer bundled in Jenkins core.
+      Provide SSHD as a plugin.
+
+  - type: major rfe
+    category: major rfe
+    pull: 5065
+    authors:
+    - timja
+    - fqueiruga
+    - josephbrueggen
+    message: |-
+      Add modern icons: build status and weather.
+
+  - type: rfe
+    category: rfe
+    pull: 5337
+    authors:
+    - jeffret-b
+    - res0nance
+    message: |-
+      Add administrative monitors recommending no executors are configured on the controller.
+
+  - type: rfe
+    category: rfe
+    pull: 5078
+    authors:
+    - daniel-beck
+    - fqueiruga
+    message: |-
+      Add indicator for security-related entries in the global administrative monitors configuration.
+
+  - type: rfe
+    category: rfe
+    pull: 5424
+    authors:
+    - daniel-beck
+    message: |-
+      Improve UI of slow trigger administrative monitor.
+
+  - type: rfe
+    category: rfe
+    pull: 5192
+    issue: 63855
+    authors:
+    - benebsiny
+    - oleg-nenashev
+    message: |-
+      Support 'min' and 'max' values in field definitions of forms.
+
+  - type: rfe
+    category: rfe
+    pull: 5392
+    authors:
+    - uhafner
+    message: |-
+      Simplify icon for unstable builds (use an exclamation mark).
```

Comment:
We normally combine multiple changelog entries in LTS right? Shouldn’t this be dropped and possibly mentioned in the other reference about modernising the icons?

Reply:
I like that idea.  That will reduce the number of entries and allow me to group the entries more clearly.

Reply:
Done in fae86747ee714609da097b4c7453a98eb4cd4436